pkg/login/social/social.go

```diff
@@ -258,8 +258,8 @@ func GetOAuthHttpClient(name string) (*http.Client, error) {
 
 	if info.TlsClientCert != "" || info.TlsClientKey != "" {
 		cert, err := tls.LoadX509KeyPair(info.TlsClientCert, info.TlsClientKey)
```

Comment:
Moved this fix to another PR https://github.com/grafana/grafana/pull/27804 since it is a bug instead of an improvement related to this task.